test/dummy_app/config/application.rb

```diff
@@ -42,7 +42,7 @@ class Application < Rails::Application
     config.assets.precompile += %w( arturo.js )
 
     config.action_controller.action_on_unpermitted_parameters = :raise if Rails.version > "4.0.0"
-    config.active_record.raise_in_transactional_callbacks = true if Rails.version > "4.2.0"
+    config.active_record.raise_in_transactional_callbacks = true if Rails.version > "4.2.0" && Rails::VERSION::MAJOR < 5
```

Comment:
This was already wrong. Lexical comparison with `"4.2.0"` is just asking for trouble. This PR is no worse at least.